.github/workflows/load-tests.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Load Tests
+
+on:
+# Starts at 2:00 AM CET
+  # schedule:
+  #   - cron: '00 2 * * *'
```

Comment:
Should this be enabled?

Reply:
Not yet. I haven't test it yet. Once it is tested we can decide weather we want to enable the pipeline to generate constant traffic.